Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -2289,6 +2289,22 @@ void Document::set_cookie(StringView cookie_string, Cookie::Source source)
     page().client().page_did_set_cookie(m_url, cookie.value(), source);
 }
 
+String Document::bg_color() const
+{
+    if (auto* body_element = body(); body_element && body_element->layout_node()) {
+        Color color = body_element->layout_node()->computed_values().background_color();
+        return color.to_string_without_alpha();
+    }
+    return ""_string;
```

Comment:
If I understand correctly, other browsers simply return the `<body bgcolor>` attribute value verbatim here, i.e something like:

```cpp
String Document::bg_color() const
{
    if (auto* body_element = body())
        return body_element->get_attribute_value(HTML::AttributeNames::bgcolor);
    return ""_string;
}

void Document::set_bg_color(String const& value)
{
    if (auto* body_element = body())
        body_element->set_attribute(HTML::AttributeNames::bgcolor, value);
}
```